tests/Behat/DrupalContext.php

```diff
@@ -306,4 +311,64 @@ protected function getLanguageIdByName(string $name): string {
     throw new \InvalidArgumentException("Language '{$name}' not found.");
   }
 
+  /**
+   * Configure language negotiation to use URL suffixes.
+   *
+   * @BeforeScenario @language-negotiation-suffix
+   */
+  public function setUpLanguageNegotiationSuffix(): void {
```

Comment:
The beforeScenario and afterScenario don't take into account the current configuration so it ends up getting lost after running the test. E.g: if I configure the site to use the suffix negotiator and run the test, after the test the site uses the prefix negotiator.